guides/common/modules/proc_synchronizing-ansible-collections.adoc

```diff
@@ -1,7 +1,12 @@
 [id="synchronizing-ansible-collections_{context}"]
 = Synchronizing Ansible Collections
 
+ifdef::satellite[]
 On {Project}, you can synchronize your Ansible Collections from Private Automation Hub, `console.redhat.com`, and other {Project} instances.
+endif::[]
+ifndef::satellite[]
+On {Project}, you can synchronize your Ansible Collections from any Ansible Galaxy and other {Project} instances.
+endif::[]
 Ansible Collections will appear on {Project} as a new repository type in the {ProjectWebUI} menu under *Content* after the sync.
 
 NOTE: Ansible Collections live in the Library environment and the lifecycle management is not supported.
```

Comment:
Is this note still relevant or should we assume for Foreman nightly and all the way down to 3.3, all Smart Proxies run Pulp3?

Reply:
Maybe @evgeni or @ekohl can answer this?